devices/HumanStateProvider/HumanStateProvider.cpp

```diff
@@ -1094,6 +1360,25 @@ bool HumanStateProvider::impl::getLinkTransformFromInputData(
     return true;
 }
 
+bool HumanStateProvider::impl::applySecondaryCalibration(
+        const std::unordered_map<std::string, iDynTree::Transform> &transforms_in, std::unordered_map<std::string, iDynTree::Transform> &transforms_out)
+{
+    transforms_out = transforms_in;
+
+    for (const auto& linkMapEntry : wearableStorage.modelToWearable_LinkName) {
+        const ModelLinkName& modelLinkName = linkMapEntry.first;
+        if (!(secondaryCalibrationRotations.find(modelLinkName)
+              == secondaryCalibrationRotations.end())) {
+            iDynTree::Rotation rotation = transforms_out[modelLinkName].getRotation();
+            rotation = rotation * secondaryCalibrationRotations.at(modelLinkName);
```

Comment:
``find`` and ``at`` are duplicate. Even if the complexity is constant, it may be better to store the iterator returned by ``find``.

Reply:
done https://github.com/robotology/human-dynamics-estimation/pull/164/commits/4544d762a2564a544717fd9bc965ba0a7d267cce